lib/dependency-manager-adapters/npm.js

```diff
@@ -7,12 +7,32 @@ var path = require('path');
 var extend = require('extend');
 var debug = require('debug')('ember-try:dependency-manager-adapter:npm');
 var rimraf = RSVP.denodeify(require('rimraf'));
+var execFileSync = require('child_process').execFileSync;
 
 module.exports = CoreObject.extend({
   init: function() {
     this._super.apply(this, arguments);
     this.run = this.run || require('../utils/run');
+    this.adapterInit();
   },
+  adapterInit: function() {
+    this.backupYarnLockPresent = fs.existsSync(path.join(this.cwd, this.yarnLockBackupFileName));
```

Comment:
Right. I thought it's better to put it here for consistency but now i see that it causes execution of code that doesn't need to be executed during setup flow. Would move to `cleanup`. Too bad this won't be that pretty =(